app/middlewares/user.js

```diff
@@ -0,0 +1,36 @@
+const { checkSchema, validationResult } = require('express-validator');
+
+exports.dataUser = (req, res, next) => {
```

Comment:
You could have a more descriptive name with the process that is carried out here.